src/Features/Core/Portable/Snippets/SnippetPlaceholder.cs

```diff
@@ -10,42 +10,42 @@ namespace Microsoft.CodeAnalysis.Snippets;
 internal readonly struct SnippetPlaceholder
 {
     /// <summary>
-    /// The identifier in the snippet that needs to be renamed.
+    /// Editable text in the snippet.
     /// </summary>
-    public readonly string Identifier;
+    public readonly string Text;
 
     /// <summary>
     /// The positions associated with the identifier that will need to
     /// be converted into LSP formatted strings.
     /// </summary>
-    public readonly ImmutableArray<int> PlaceHolderPositions;
+    public readonly ImmutableArray<int> StartingPositions;
```

Comment:
Wanted to emphasize that these are starting positions for clarity. Also since they are already inside `SnippetPlaceholder` type, having `PlaceHolder` in the name is an overkill